core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3529,21 +3553,39 @@ private SqlDialect nonOrdinalDialect() {
         + "from \"foodmart\".\"employee\" A "
         + "join \"foodmart\".\"department\" B\n"
         + "on A.\"department_id\" = B.\"department_id\"";
-    final String expected = "SELECT *\n"
+    final String expected = "SELECT employee.employee_id, employee.full_name, employee.first_name, "
+        + "employee.last_name, employee.position_id, employee.position_title, employee.store_id, "
+        + "employee.department_id, employee.birth_date, employee.hire_date, employee.end_date, "
+        + "employee.salary, employee.supervisor_id, employee.education_level, "
+        + "employee.marital_status, employee.gender, employee.management_role, "
+        + "department.department_id AS department_id0, department.department_description\n"
         + "FROM foodmart.employee AS employee\n"
-        + "INNER JOIN foodmart.department AS department "
-        + "ON employee.department_id = department.department_id";
```

Comment:
this diff (and a few others below) is spurious. you've just moved `ON` from one line to the next. please revert to the original formatting.